src/main/java/com/uid2/shared/cloud/CloudStorageS3.java

```diff
@@ -46,6 +47,25 @@ public CloudStorageS3(String accessKeyId, String secretAccessKey, String region,
         this.bucket = bucket;
     }
 
+    public CloudStorageS3(String region, String bucket, String endpoint) {
```

Comment:
Is there a reason the parameter is called endpoint, but the other overload has one called s3Endpoint? Are they the same thing?

Reply:
No, I just thought it's a better name, given the whole class is already about S3. Will change it for consistency though.